composer.json

```diff
@@ -28,7 +28,8 @@
         "atrox/haikunator": "1.*",
         "paragonie/halite": "^4",
         "phpoffice/phpspreadsheet": "^1.24",
-        "phpmailer/phpmailer": "6.*"
+        "phpmailer/phpmailer": "6.*",
+        "ext-fileinfo": "*"
```

Comment:
Do we need `ext-fileinfo`? It should be included with php >= 5.1 and we run `PHP 8.1.12-1ubuntu4.3.` I also tested it and it works without the dependency

Reply:
not needed, will remove